multiple_generate.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+while getopts s:e:u: option
+do
+ case "${option}"
+ in
+ s) SUBPROJECT=${OPTARG};;
+ e) EMAIL=${OPTARG};;
+ u) UNIQUEID=${OPTARG};;
+ esac
+done
+
+function clean() {
+  rm -rf $BUILD_DIR
+}
+
+BASE=$(pwd)
+
+if [ "${WEBUI:-false}" == "true" ]; then
+  BUILD_DIR=$(pwd)/temp/$EMAIL/$UNIQUEID
+  mkdir -p $BUILD_DIR
+  cd $BUILD_DIR
+else
+  git clone https://github.com/fossasia/yaydoc.git yaydocclone
+  BASE=$(pwd)/yaydocclone
+  BUILD_DIR=$(pwd)/temp
+  echo -e "build dir"
+  echo -e $BUILD_DIR
+  mkdir -p $BUILD_DIR
+  cd $BUILD_DIR
+fi
+
+
+IFS=', ' read -r -a SUBPROJECTS <<< "$SUBPROJECT"
+
+for element in "${SUBPROJECTS[@]}"
+do
+  URL_SPLIT=(${element//// })
+  REPONAME=(${URL_SPLIT[3]//./ })
+  git clone $element ${REPONAME}_temp &
+done
+
+wait
+
+for element in "${SUBPROJECTS[@]}"
+do
+  URL_SPLIT=(${element//// })
+  REPONAME=(${URL_SPLIT[3]//./ })
+  cd ${REPONAME}_temp
+  mkdir ../$REPONAME
+  cp -a docs/. ../$REPONAME/.
+  cd ..
+  rm -rf ${REPONAME}_temp
+done
+
+cp -a ${BASE}/modules/scripts/ $BUILD_DIR/
+cp -a ${BASE}/modules/templates/ $BUILD_DIR/
+
+mkdir _themes
+
+cp -a ${BASE}/fossasia_theme $BUILD_DIR/_themes/
+
+URL_SPLIT=(${SUBPROJECTS[0]//// })
```

Comment:
Yeah, I made this script by having web UI on the mind. Once it gets merged. we'll raise a separate issue for Travis by that time I'll work on #169. That's why I didn't touch `generate_ci.sh`. if you want me to make changes for Travis in this PR also no problem. I'll make necessary changes.